partner_design_mockup/__manifest__.py

```diff
@@ -1,6 +1,6 @@
 {
     'name': 'Partner Design Mockup',
-    'version': '12.0.0.5',
+    'version': '12.0.0.6',
```

Comment:
I would argue that this is an addition that enduser would see (adding dropdowns and a button) so the version number should be 12.0.0.1.0 atleast

Reply:
Hi @nils-vertel I updated now.